### PR TITLE
Rewrite to grid

### PR DESCRIPTION
**Pull request by vrugtehagel**
Took me a while to get back to it, but here it is: progress!

So for this PR I'm introducing grid to lay out the individual items in order to prepare for some responsiveness improvements. I'm mainly aiming to get rid of the margin-padding-float based layout because that's a bit of a pain to make work on smaller screens. There are still some paddings and margins left to fix, but that'll be something for another PR.

This is not intended to make any visual changes (although it does change things very slightly). Anyhow, here's some screenshots. At an around 1330px wide screen (before top, after on the bottom):

image
image

And on a 1000px wide screen, which is about as small as it gets without losing usability:

image
image

### CODE REVIEW

File: style.css
Comment (vrugtehagel) [inline style.css]: Not sure if you'd want them together with their other declarations or this, so let me know what you'd prefer here

File: style.css
Comment (vrugtehagel) [inline style.css:353]: I don't usually write my CSS this way, so I'm not entirely sure what would be desirable in this case (either having this empty block or just unindenting all of its sub-blocks)

Comment (LeaVerou) [reply style.css:353]: This is fine

File: style.css
Comment (LeaVerou) [reply style.css]: I'd prefer them together with their other declarations if that's ok

**Review by LeaVerou — APPROVED** [review]
Thank you and happy new year! Left some replies to your comments, other than that LGTM. I’ll approve now so you can merge whenever you get around to making the changes.

**Comment by vrugtehagel** [comment]
Thanks! Happy new year to you too 🙂 

I made some changes to address your comments - again wasn't entirely sure if I should be indenting stuff or not, so feel free to add more comments if things look off to you.

Also, I can't merge this even with an approval, because I don't have write access to this repo. You'll have to merge it yourself, I'm afraid 😉

**Comment by LeaVerou** [comment]
Merged, thanks!

**Pull request merged**
